app/models/pulitzer/content_element.rb

```diff
@@ -34,6 +45,37 @@ def empty_body?
       image_type? ? !image? : body.blank?
     end
 
+    def clone_me
+      clone_attrs = self.attributes.dup
+      clone_attrs.delete 'id'
+      clone_attrs.delete 'version_id'
+
+      type_clone_method = 'clone_' + type.to_s
+      if respond_to? type_clone_method
+        my_clone = send type_clone_method, clone_attrs
```

Comment:
@gomayonqui `public_send` is intended to be used by callers who should not have access to private methods. `send` is intended to be used by callers who should have access to private methods. When I call a method on a model from within that same model, `send` is the appropriate call, since I should have access to all private methods. If I called this method from some other model, then I should use `public_send`.